dynamiqs/utils/wigners.py

```diff
@@ -45,17 +45,29 @@ def wigner(
     """
     state = jnp.asarray(state)
 
-    if state.ndim > 2:
-        raise NotImplementedError('Batching is not yet implemented for `wigner`.')
-
     rdtype = dtype_complex_to_real(state.dtype)
     xvec = jnp.linspace(-xmax, xmax, npixels, dtype=rdtype)
     yvec = jnp.linspace(-ymax, ymax, npixels, dtype=rdtype)
 
     if method == 'clenshaw':
         state = todm(state)
-        w = _wigner_clenshaw(state, xvec, yvec, g)
+        expanded = False
+        if state.ndim == 2:
+            state = state[None, ...]
+            expanded = True
```

Comment:
Do we really need to do this every time we want to batch over a potentially non-existing dimension?  (especially having to remember `expanded` is a bit annoying) Is there a shorter way/trick/one liner/utility function/context manager we could use?

Reply:
I'm open to any idea

Reply:
I'll think about it 😊